MAINTAINERS

```diff
@@ -508,3 +510,7 @@ M: Ira Cooper
 M: Shwetha Panduranga
 M: Nithya Balachandran
 M: Raghavendra Gowdappa
+M: Susant Palai
+M: Barak Sason Rofman <bsasonro@redhat.com>
+M: Tamar Shacked <tshacked@redhat.com>
+M: Csaba Henk <chenk@redhat.com>
```

Comment:
This section is for people who stopped contributing.

Reply:
Done